ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -202,6 +213,53 @@ access_lists:
         action: "< action as string >"
 ```
 
+Improved data model has a more sophisticated design documented below:
+
+```yaml
+ip_access_lists:
+  - name: "< access list name as string >"
+    counters_per_entry: < true | false >
+    entries:
+      # remark entry
+      - sequence: < acl entry sequence number >  # optional
+        # NOTE: if remark is defined, other keys in acl entry will be ignored
+        remark: "< Comment, up to 100 characters >"
+      # normal entry
+      - sequence: < acl entry sequence number >  # optional
+        action: "< permit | deny >"  # required
+        protocol: "< ip | tcp | udp | icmp | other protocol name or number >"  # required
+        # NOTE: A.B.C.D without a mask means host
+        source: "< any | A.B.C.D/E | A.B.C.D >"  # required
```

Comment:
If we support ```source: < A.B.C.D >``` then we need another knob to provide the ```Destination Wildcard bits```. Same for the destination.

Reply:
The code automatically inserts `host` before the IP if only `A.B.C.D` is given.

Reply:
@tgodaA It's also documented in the doc and IMO makes the data model easier, at least for humans. Suggest closing this.

Reply:
Sorry, I didn't see the option ```host A.B.C.D``` assuming mask is a must. My bad!